src/presentation/styles/global.pcss

```diff
@@ -0,0 +1,3 @@
+:global(body) {
```

Comment:
maybe we should use [normalize.css](https://necolas.github.io/normalize.css/) instead?